src/styles/blocks/header.scss

```diff
@@ -0,0 +1,53 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.logo__img {
+  display: flex;
```

Comment:
The `.logo__img` class should not have `display: flex;` unless there is a specific reason for it. Typically, images do not need to be set as flex containers.